sink/sink.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+    "bufio"
+    "fmt"
+    "github.com/prometheus/client_golang/prometheus"
+    "github.com/prometheus/client_golang/prometheus/promauto"
+    "github.com/prometheus/client_golang/prometheus/promhttp"
+    "net"
+    "net/http"
+    "strconv"
+    "strings"
+    "time"
+)
+
+var (
+        metricsInjectorReceived = promauto.NewCounter(prometheus.CounterOpts{
+                Name: "sink_metrics_injector_received_count",
+                Help: "The total number of metrics received from injector",
+        })
+        metricsCarbonReceived = promauto.NewCounter(prometheus.CounterOpts{
+                Name: "sink_metrics_carbon_received_count",
+                Help: "The total number of metrics received from carbon relay",
+        })
+        metricsLatency = promauto.NewHistogram(prometheus.HistogramOpts{
+                Name:    "sink_metrics_latency",
+                Help:    "Histogram latency metrics", 
+		Buckets: prometheus.ExponentialBuckets(10000000,10,3),
+        })
+)
+
+func getLatency(epochTimeStamp int64) int64 {
```

Comment:
This function is not latency-specific. It returns an elapsed duration between a time reference and "now".

What about naming it `getElapsedTimeSince`?